draft-ietf-privacypass-architecture.md

```diff
@@ -544,6 +545,76 @@ and guidelines on how to deploy the protocol to minimize any privacy impacts.
 Any extension to the Privacy Pass protocol MUST adhere to the guidelines
 specified in {{issuer-role}} for managing Issuer public key data.
 
+## Information Flow {#flow}
+
+The end-to-end run of redemption and issuance protocol involves information
+flowing between Issuer, Origin, Client, and Attester. That information can
+have implications on the privacy goals that Privacy Pass aims to provide
+as outlined in {{privacy-and-trust}}. In this section, we describe the flow
+of information between each party. How this information affects the privacy
+goals in particular deployment models is further discussed in {{deployment}}.
+
+### Token Challenge Flow {#challenge-flow}
+
+To use Privacy Pass, Origins choose an Issuer from which they are willing to
+accept tokens. Origins then construct a token challenge using this specified
+Issuer and the redemption context it shares with the Client. This token
```

Comment:
I think we need to keep redemption context here, since that's the information the Origin has available to it to construct a token challenge. I like the other suggestion to explain why issuance secrecy is important.

Reply:
I still don't understand whether "redemption context" refers to data or something else though.

Reply:
It refers to _anything that the origin can see during redemption_ (defined earlier in the doc).

Reply:
(Unresolving) @martinthomson does the improved description in #343 work?

Reply:
```suggestion
Issuer and information from the redemption context it shares with the Client. This token
```